api/api.yaml

```diff
@@ -2020,14 +2020,9 @@ paths:
                   description: The type of the destination.
                   example: webhook
                   enum: [webhook, kinesis]
-                provider:
-                  type: string
-                  description: The provider of the destination.
-                  example: svix
-                  enum: [svix, outpost]
                 secrets:
                   type: object
-                  description: Secrets for the destination. This is only used for Kinesis destinations.
+                  description: Secrets for the destination.
```

Comment:
Removing the mention of kinesis so this comment doesn't become outdated when we support other destinations that need secrets